plugins/jobs/server/models/job.py

```diff
@@ -52,19 +56,33 @@ def _validateStatus(self, status):
             raise ValidationException(
                 'Invalid job status %s.' % status, field='status')
 
-    def list(self, user=None, limit=0, offset=0, sort=None, currentUser=None):
+    def list(self, user, types=None, statuses=None,
```

Comment:
Why do you think `'none'` is better than `None`?, I want to understand your thinking here.

For me `None` seems fine, because IIUC `user` says which jobs you want based on which users are attached to the job.  `None` means anonymous jobs, `'all'` means all users, and a specific user document gets you jobs tied to that user.  

If you prefer to include `'none'`, I'm ok with that, you have definitely thought about this more than me, as long as the interface goes back to `user=None` as default.